doc/xanadu_theme/header.html

```diff
@@ -61,4 +64,4 @@
   </div>
   <!-- Collapsible content -->
 </nav>
-<!--/.Navbar-->
\ No newline at end of file
+<!--/.Navbar-->

```

Comment:
Not sure about this :thinking: was updating the file online :thinking:

Reply:
All good!